### ISSUE TITLE
PNG support

### ISSUE THREAD

**Issue by Flexmaen** [darktrojan/shrunked#25]
Great plugin, but PNG files will be ignored (at least on Seamonkey 2.49.2).

**Comment by mMuck** [comment]
Yes this would be great. I work often with copy and pasted inline images which mostly are PNG ones I need to resize externally. Would be cool to do the PNG to resized JPEG transition too, perhaps interactive or predefined profile based at this footer bar to select with one click at a button a profile like PNG 500px, JPG 800px, ...

@darktrojan As you mentioned you probably won't find the time. Hopefully someone can push that great (and still working) add-on to a fully-funcional image resizer like there was one before.

BTW: Can you put labels like "feature request", "help needed", "bug" and "needs information" to give a better overview at the issue list? Thx

**Comment by Tragen** [comment]
Same here. I tested the plugin and it didn't work until I realized it works only for jpegs.
Before I used Auto Resize Image but it's not available anymore for newer TB versions.

**Comment by petr-ekl-itcube** [comment]
Yes, definitely PNG, plase... I had the same experience as Tragen
